File: Intro to C-Sharp (CIS162AD)/03 Methods/ScopeExample.cs
/*
 * This example demonstrates variable scope. Because it includes references to out-of-scope variables, this code will not compile.
 */
public class ScopeExample {

    public static void Main(String[] args) {
        int numberOfSongs = 7;
        Console.WriteLine(numberOfSongs); // no problem!

        OutputNumber();
    }

    public static void OutputNumber() {
        Console.WriteLine(numberOfSongs); // favoriteNumber doesn't exist here!
    }
}

File: OOP with Java (CIS150AB)/03 Methods/ScopeExample.java
/*
 * This example demonstrates variable scope. Because it includes references to out-of-scope variables, this code will not compile.
 */
public class ScopeExample {

    public static void main(String[] args) {
        int numberOfSongs = 7;
        System.out.println(numberOfSongs); // no problem!

        outputNumber();
    }

    public static void outputNumber() {
        System.out.println(numberOfSongs); // favoriteNumber doesn't exist here!
    }
}
